code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -574,3 +574,198 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 			user << "<span class='warning'>You need to dry this first!</span>"
 	else
 		..()
+
+///////////////
+//VAPE NATION//
+///////////////
+/obj/item/clothing/mask/vape
+	name = "e-cigarette"
+	desc = "A classy and highly sophisticated electronic cigarette, for classy and dignified gentlemen. A warning label reads \"Warning: do not fill with flamable materials!\""//<<< i'd vape to that.
+	icon = 'icons/obj/clothing/masks.dmi'
+	icon_state = null
+	item_state = null
+	var/chem_volume = 100
+	var/vapetime = 0 //this so it won't puff out clouds every tick
+	var/screw = 0 // kinky
+	var/super = 0 //for the fattest vapes dude.
+	var/emagged = 0 //LET THE GRIEF BEGIN
+
+/obj/item/clothing/mask/vape/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	var/obj/item/organ/brain/B = H.getorgan(/obj/item/organ/brain)
+	user.visible_message("<span class='suicide'>[user] starts to blow into [src] instead of sucking on it! Oh no, what are they doing?")
+	sleep(5)
+	playsound(loc, 'sound/effects/pop_expl.ogg', 50, 1)
+	add_fingerprint(user)
+	if (H && !qdeleted(H))
+		if (B && !qdeleted(B))
+			H.internal_organs -= B
+			qdel(B)
```

Comment:
Also, you cannot remove an organ directly from the list like that. You must use B.Remove(H).